servers/oidc.py

```diff
@@ -1,14 +1,16 @@
 """Mock OAUTH2 aiohttp.web server."""
 
 from aiohttp import web
-from authlib.jose import jwt, JsonWebKey
+from joserfc import jwt
+from joserfc.jwk import ECKey, RSAKey, KeySet
 from typing import Tuple, Union
+import json
 import ssl
 import sys
 from pathlib import Path
 
 HTTP_PROTOCOL = "http"
-OIDC_PORT = 8080
```

Comment:
This repository is not meant for development, and this particular mock service is only so that the demo will work without involving components form other repositories.

Reply:
Still don't see why bake it like this but no strong opinion about it.